java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -239,11 +239,12 @@ public interface ListBaseCommands {
      *     href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands">Blocking
      *     Commands</a> for more details and best practices.
      * @param keys The <code>keys</code> of the lists to pop from.
-     * @param timeout The number of seconds to wait for a blocking <code>BLPOP</code> operation to
-     *     complete. A value of <code>0</code> will block indefinitely.
-     * @return An <code>array</code> containing the <code>key</code> from which the element was popped
-     *     and the <code>value</code> of the popped element, formatted as <code>[key, value]</code>.
-     *     If no element could be popped and the timeout expired, returns </code>null</code>.
+     * @param timeout The number of seconds to wait for a blocking operation to complete. A value of
```

Comment:
@Yury-Fridlyand everything else is minor and looks good.  
Please pull out the BLPOP and BRPOP documentation update into a separate PR.  Textual changes require more review and should be done separately.

Reply:
See #201